CIPs/0002.md

```diff
@@ -0,0 +1,93 @@
+# CIP 2: Professional Verifier Pool
```

Comment:
Please ensure your title follows the new cip-template.md file.

Also ensure status is Draft.

https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also please ensure the rest of the document headings are following the new CIP template document. Abstract, Simple Summary, Motivation, Rationale, Specification, CopyRight, Implementation, etc.